libraries/AP_Motors/examples/AP_Motors_test/wscript

```diff
@@ -2,6 +2,9 @@
 # encoding: utf-8
 
 def build(bld):
+
+    bld.env.DEFINES.append("AP_MOTORS_DENSITY_COMP=0")
```

Comment:
This is currently failing CI because this define is being used in the main build, being here I would have expected it to only be used in the example.

Reply:
Moved to using build type in the cpp